packages/testbed/src/app/pages/fx-settings/fx-settings.component.html

```diff
@@ -0,0 +1,16 @@
+@for (bus of buses; track bus) {
+<div class="mb-5">
+  <h2>
+    {{ bus.label }}
+    <span class="small text-muted"
+      >{{ withFxType(bus.fx.fxType$) | async }} ({{ bus.fx.fxType$ | async }})</span
+    >
+  </h2>
+  <div>
+    @for (v of [1, 20, 60, 80, 100, 220, 400, 1000]; track v) {
```

Comment:
Same here, if it's just to show the usability, I'm ok with buttons, but i would make a number input so the user can choose whatever he wants

Reply:
Yes, this is just for testing ;-)